profile_page/database/profile_page_db.py

```diff
@@ -1,11 +1,11 @@
-from pymongo import MongoClient
+from pymongo import MongoClient, ReturnDocument
```

Comment:
Had some issues with the test that updates the graph and game. I updated them using array_filters but they do not fail when the conditions dont find the filter. For example, if the user passed an ID that didnt exist in the user's profile, it would just pass back the return as if the update was successful, even if no changes were made. I added the return document to be able to send back the current state of the document after the update, and then this allowed me to compare the old user profile with the new user profile in the tests. It\f they were the same then the test passing the data of a non-existant game would pass. See lines 143, 228 in profile_page_db.py , and lines 122, 138 in test_db.py